a3p-integration/proposals/n:upgrade-next/synthetic-chain-excerpt.js

```diff
@@ -57,7 +59,6 @@ const agdBinary = 'agd';
 
 /**
  * @param {{execFileSync: typeof import('child_process').execFileSync }} io
- * @returns
```

Comment:
This one I just omitted, so TS would infer the return type which is more complicated than I wanted to write down.